src/cogs/gamble.py

```diff
@@ -0,0 +1,152 @@
+            embed.add_field(name=f'{amount_to_bet}$ has been added to your wallet', value=f'You have {amount_result}$')
+            await ctx.send(embed=embed)
+        else:
+            embed = discord.Embed(title='You lost.', color=discord.Color.red())
+            embed.add_field(name=f'{amount_to_bet}$ has been removed from your wallet', value=f'You have {amount_result}$')
+            await ctx.send(embed=embed)
+    
+    @commands.hybrid_command(with_app_command=True)
+    async def wallet(self, ctx: Context, user: Member = None):
+        """
+        Check your wallet. (Optional) Enter another user's ID to view their wallet.
+        """
+        if user is None:
+            user = ctx.author
+
+        user_database_check_if_user_exists_otherwise_add(user.id)
+
+        embed = discord.Embed(title=f'{user.display_name}\'s wallet', color=discord.Color.blurple())
+        embed.set_thumbnail(url='https://cdn.iconscout.com/icon/free/png-512/free-wallet-588-456730.png')
+        embed.add_field(name='Money', value=f'{user_database_get("money", user.id)}$')
+
+        await ctx.send(embed=embed)
+    
+    @commands.hybrid_command(with_app_command=True)
+    async def daily(self, ctx: Context):
+        """
+        Get a small amount of money, works once per day.
+        """
+        user_database_check_if_user_exists_otherwise_add(ctx.author.id)
+
+        last_daily_timestamp = int(user_database_get('last_daily_timestamp', ctx.author.id))
+        current_timestamp = int(time.time())
+        seconds_in_a_day = 86400
+        seconds_since_last_daily = current_timestamp - last_daily_timestamp
+        if seconds_since_last_daily < seconds_in_a_day:
+            formatted_time = datetime.fromtimestamp(seconds_in_a_day - seconds_since_last_daily).strftime('%Hh %Mm %Ss')
+            await ctx.send(f'You need to wait {formatted_time} before you can use this again.', ephemeral=True)
+            return
+        
+        # Update timestamp and add money to user's wallet.
+        user_database_set('last_daily_timestamp', current_timestamp, ctx.author.id, commit_changes=False)
+        user_database_set('money', daily_amount, ctx.author.id, commit_changes=True, add_value_instead=True)
+
+        await ctx.send(f'{ctx.author.mention} {daily_amount}$ has been added to your wallet.')
+    
+    @commands.hybrid_command(with_app_command=True)
+    async def give(self, ctx: Context, user: Member, amount: int):
+        """
+        Transfer money to another member. First argument is user ID. Ssecond argument is amount.
+        """
+        user_database_check_if_user_exists_otherwise_add(ctx.author.id)
+
+        if amount <= 0:
+            await ctx.send('You must enter a positive number greater than 0.', ephemeral=True)
+            return
+        if amount > user_database_get('money', ctx.author.id):
+            await ctx.send('You don\'t have enough money.', ephemeral=True)
```

Comment:
Rather than escaping you can use `"You don't have enough money."`. We've done this pretty arbitrarily so far but ig for longer strings double quotes should be preferred